network-modification/src/main/java/com/powsybl/network/modification/tripping/AbstractInjectionTripping.java

```diff
@@ -4,32 +4,34 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package com.powsybl.contingency.tasks;
+package com.powsybl.network.modification.tripping;
 
-import com.powsybl.computation.ComputationManager;
-import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.Injection;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.Switch;
+import com.powsybl.iidm.network.Terminal;
 
 import java.util.Objects;
 import java.util.Set;
 
 /**
  * @author Teofil Calin BANC <teofil-calin.banc at rte-france.com>
  */
-public abstract class AbstractInjectionTripping extends AbstractTrippingTask {
+public abstract class AbstractInjectionTripping extends AbstractTripping {
 
     protected final String id;
 
-    public AbstractInjectionTripping(String id) {
+    protected AbstractInjectionTripping(String id) {
```

Comment:
I'm not really in favour of restricting previously public classes as we can not be sure it has not been used elsewhere.

Reply:
yes, indeed. I'll revert that. And maybe we should also change that sonar rule.

Reply:
Sonar rules is ok. abstract class should not be exposed....